modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/seedlist/types.scala

```diff
@@ -1,20 +1,49 @@
 package org.tessellation.sdk.infrastructure.seedlist
 
+import cats.Order
+import cats.syntax.contravariant._
+import cats.syntax.option._
+
 import org.tessellation.schema.peer.PeerId
-import org.tessellation.security.hex.Hex
+import org.tessellation.schema.trust._
 
-import fs2.data.csv.RowDecoder
+import fs2.data.csv._
 import fs2.data.csv.generic.semiauto.deriveRowDecoder
 
 object types {
 
-  case class SeedlistCSVEntry(id: String) {
+  case class SeedlistEntry(
+    peerId: PeerId,
+    ipAddress: Option[String],
```

Comment:
you can use a dedicated type for IP address - `Host` or `IpAddress` from `com.comcast.ip4s`

Reply:
The IP Address has been changed to `IpAddress`.